sidekick/views.py

```diff
@@ -1,8 +1,71 @@
+import requests
+import json
 from os import environ
 from django.http import JsonResponse
+from rest_framework import status
+
+from .models import Organization
 
 
 def health(request):
     app_id = environ.get("MARATHON_APP_ID", None)
     ver = environ.get("MARATHON_APP_VERSION", None)
     return JsonResponse({"id": app_id, "version": ver})
+
+
+def send_wa_template_message(request):
+    data = request.GET.dict()
+
+    required_params = [
+        "org_id",
+        "wa_id",
+        "namespace",
+        "element_name",
+        "message",
+    ]
+    for field in required_params:
+        if field not in data:
+            return JsonResponse(
+                {
+                    "error": "Required fields: {}".format(
+                        ", ".join(required_params)
+                    )
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+    org_id = data["org_id"]
+    wa_id = data["wa_id"]
+    namespace = data["namespace"]
+    element_name = data["element_name"]
+    message = data["message"]
```

Comment:
There might be a [django View class](https://docs.djangoproject.com/en/2.1/ref/class-based-views/) that we can use, this feels like a fairly common pattern. Otherwise, I'm wondering whether it might be prudent to create a View Class Mixin where you just declare the `required_params`. However this is just me musing, so feel free to ignore this one 😅